tools/rosbag_storage/include/rosbag/bag.h

```diff
@@ -111,6 +112,17 @@ class ROSBAG_DECL Bag
 
     ~Bag();
 
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
+    Bag(Bag&& other) {
+        init();
+        swap(other);
+    }
+    Bag& operator=(Bag&& other) {
+        swap(other);
+        return *this;
+    }
```

Comment:
Please keep the implementation of these methods in the cpp file even though they are short.